diff --git a/tests/Makefile.am b/tests/Makefile.am
index 11a083c63..17c26bcb0 100644
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -505,7 +505,7 @@ dtls_stress_SOURCES = dtls/dtls-stress.c
 dtls_stress_LDADD = $(COMMON_GNUTLS_LDADD) \
 	$(COMMON_DEPS_LDADD)
 
-dist_check_SCRIPTS += fastopen.sh pkgconfig.sh starttls.sh starttls-ftp.sh starttls-smtp.sh \
+dist_check_SCRIPTS += fastopen.sh starttls.sh starttls-ftp.sh starttls-smtp.sh \
 	starttls-lmtp.sh starttls-pop3.sh starttls-xmpp.sh starttls-nntp.sh starttls-sieve.sh \
 	ocsp-tests/ocsp-tls-connection ocsp-tests/ocsp-must-staple-connection \
 	ocsp-tests/ocsp-test cipher-listings.sh sni-hostname.sh server-multi-keys.sh \
