--- tests/status-request-revoked.c.orig	2020-01-18 05:50:33.000000000 -0800
+++ tests/status-request-revoked.c	2020-11-18 10:03:44.000000000 -0800
@@ -65,6 +65,16 @@
 	fprintf(stderr, "client|<%d>| %s", level, str);
 }
 
+static time_t mytime(time_t * t)
+{
+	time_t then = 1586000000;
+
+	if (t)
+		*t = then;
+
+	return then;
+}
+
 static unsigned char server_cert_pem[] =
 "-----BEGIN CERTIFICATE-----\n"
 "MIIEKjCCAhKgAwIBAgIIRiBQA6KFBj0wDQYJKoZIhvcNAQELBQAwDzENMAsGA1UE\n"
@@ -252,6 +262,7 @@
 	unsigned int status;
 	int ret;
 
+	gnutls_global_set_time_function(mytime);
 	ret = gnutls_certificate_verify_peers2(session, &status);
 	if (ret < 0)
 		return -1;
